### DISCUSSION TITLE
Hide duplicate track if more than one file has been found

### DISCUSSION THREAD

**Discussion by WAPEETY** [navidrome/navidrome#3067] [Ideas]
If I have the same exact track (maybe based on the exact title and album) would be great to hide the duplicated song from the album:

Example:
![image](https://github.com/navidrome/navidrome/assets/38016769/ae46ae80-09ef-43e2-88b3-b8137e06314d)

**Comment by deluan** [comment]
Why not remove the duplicate? If the file appears twice in your library, Navidrome should not decide which one to display. 

I think the sensible thing to do is show all files in your library so you can take action on this.

**Reply by WAPEETY** [reply]
yes but wouldn't be nice to, have an automatic "hide duplicates" option?

**Reply by WAPEETY** [reply]
PS: I'm aware this would not be a priority

**Reply by deluan** [reply]
How would Navidrome would decide which copy to remove? I think this has to be done by the user. 

IMO, what is missing is a way to "hide" tracks using the UI, similar to adding a `.ndignore` file but easier to use.

**Reply by WAPEETY** [reply]
> How would Navidrome would decide which copy to remove? I think this has to be done by the user.
> 
> IMO, what is missing is a way to "hide" tracks using the UI, similar to adding a `.ndignore` file but easier to use.

exactly what I meant, maybe the example with the file was not clear.
